File: Tanmay Jaiswal/day-11/11b.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int i=0;
        int n = nums.size();

        int ans = 0;
        int count = 0;

        for (int i=0; i<n; i++) {
            if (nums[i]) count++;
            else count = 0;

            ans = max(ans, count);
        }

        return ans;
    }
};

File: Mahek Jhamb/Day 11/11B.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: list[int]) -> int:
        count=0
        maxC=0
        for i in nums:
            if i==1:
                count+=1
                maxC=max(maxC,count)
            else :
                count=0
        return maxC